websites/M/Manacube/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+      "name": "Jackelele",
+      "id": "377175513372033024"
+    },
+    "service": "Manacube",
+    "description": {
+      "en": "An amazing Minecraft Network"
+    },
+    "url": "manacube.com",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/uwd27zO.png",
+    "thumbnail": "https://i.imgur.com/sBgHblJ.png",
```

Comment:
Thumbnail should be a promotional banner or a screenshot of the website homepage.